project/pimcore.py

```diff
@@ -0,0 +1,14 @@
+from .remote import RemoteProject
+
+
+class Pimcore5(RemoteProject):
+    major_version = 'v5.8.2'
+    remote = 'https://github.com/pimcore/pimcore.git'
+
+    @property
+    def platformify(self):
+        return ['cd {0} && rm -rf .ci .travis .gitlab-ci.yml .travis.yml .github CONTRIBUTING.md'.format(self.builddir),
```

Comment:
We don't strip these files in any other repo.  No need to do so here.  If we were to start doing that we should do it in some more centralized and automated way.